apps/sms/tools/update_nga_libs.sh

```diff
@@ -9,10 +9,15 @@
 ###############################################################################
 # variables
 bower_path="bower_components"
+
 threads_repo="gaia-components/threads#master"
 threads_index="$bower_path/threads/index.js"
-threads_name="bridge"
-threads_path="lib/$threads_name.js"
+threads_plugins="$bower_path/threads/lib/plugins"
```

Comment:
Well I'm that good at shell scripts :) So will be happy to hear any recommendations/objections!